### ISSUE TITLE
Notification on new movie release

### ISSUE THREAD

**Issue by villieb** [Radarr/Radarr#5801]
**Is your feature request related to a problem? Please describe.**
Looking at the current notifications (for email / telegram etc...) a notification can be sent out 'on grab', 'on import' and 'on upgrade', and this works perfectly and expected (especially on Sonarr).

For movies, I like to browse through the various releases before choosing one. I would like to get notified when a movie is released and available.

**Describe the solution you'd like**
From RADARR, an additional notification trigger that would send a notification when a movie has been released (i.e. it's release date). The release date is available from TMDb - could this be used to send triggers?

**Comment by RockZors** [comment]
Some of these notifications can be handled via [NZB360](https://play.google.com/store/apps/details?id=com.kevinforeman.nzb360&hl=en_US&gl=US)

**Comment by austinwbest** [comment]
> Some of these notifications can be handled via [NZB360](https://play.google.com/store/apps/details?id=com.kevinforeman.nzb360&hl=en_US&gl=US)

I'm guessing radarr in all caps is a hint to offer it natively and not via 3rd party apps. Just a guess though

**Comment by austinwbest** [comment]
> **Is your feature request related to a problem? Please describe.**
> Looking at the current notifications (for email / telegram etc...) a notification can be sent out 'on grab', 'on import' and 'on upgrade', and this works perfectly and expected (especially on Sonarr).
> 
> For movies, I like to browse through the various releases before choosing one. I would like to get notified when a movie is released and available.
> 
> **Describe the solution you'd like**
> From RADARR, an additional notification trigger that would send a notification when a movie has been released (i.e. it's release date). The release date is available from TMDb - could this be used to send triggers?
> 

Especially on sonarr, so that leads me to wonder what doesn't work with radarr notifications to warrant making that distinction?

**Comment by villieb** [comment]
That's right, I logged this feature request to provitlde a native option to have some notification that a release has been made available based on a movie release date.

I mentioned sonarr because I use notifications in sonaar more than I use in radarr. There's nothing to wonder :)

**Comment by villieb** [comment]
radarr notification triggers do not show any for movie release.

(Almost forgot about this)

**Comment by austinwbest** [comment]
> radarr version 4.1.0.6175 notification triggers do not show any for movie release.
> 
> (Almost forgot about this)

Without looking at versions, if you are on master you won't see it yet.
